### DISCUSSION TITLE
migrate asdf-plugin datree here

### DISCUSSION THREAD

**Discussion by fredleger** [asdf-community/.github#42] [General]
repo: https://github.com/webofmars/asdf-plugin-datree

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)

**Reply by fredleger** [reply]
hi @smorimoto will try to find time to dive into it
thxs for the feedback and the tools ;-)

**Reply by smorimoto** [reply]
Cool, I just published the guidelines for that. We are automating everything to make all processes transparent. You can probably do it relatively easily.

**Reply by smorimoto** [reply]
https://github.com/asdf-community/.github/blob/master/GUIDELINES.md
